contrib/hawq-docker/Makefile

```diff
@@ -227,3 +229,85 @@ distclean:
 		docker network rm $(NETWORK) 2>&1 >/dev/null || true; \
 	fi
 	@echo "Distclean Done!"
+
+hawq:
```

Comment:
It's better to add make dependencies for this option, or add description what's the make sequence to call this option.

Reply:
@radarwave agreed and fixed